python/cudf/cudf/core/multiindex.py

```diff
@@ -1423,14 +1423,14 @@ def _clean_nulls_from_index(self):
 
     def memory_usage(self, deep=False):
```

Comment:
Can you add the same warning that we have for the `deep` parameter in `DataFrame.memory_usage` since the parameter is ignored?